Deliverables/first

dhorki opened the pull request:
First deliverable
Component and CSS structure for topbar, sidebar and gallery

vinfante committed:
First deliverable
Component and CSS structure for topbar, sidebar and gallery

vinfante committed:
Adding docs folder

vinfante committed:
Index fix

vinfante committed:
Included sass

vinfante committed:
Fixed slider animation

ChrisAlonsoA commented:
Hola @dhorki ! 

Este primer entregable te quedo super bien! Implementaste todo los puntos necesarios. Me gusto mucho tu implementación de Styled-Components y que la app es Responsive. Tu código esta limpio y usas buenas prácticas. Felicidades! 

### Observaciones: 

- Te recomiendo separar los estilos en un archivo aparte y luego importarlos en tu componente.
- Una buena práctica es separar páginas de componentes, pudieras crear un Folder llamado pages. (Recordando que una página es un conjunto de componentes) .
- La parte de Grid de videos puede estar centrada.

### Bonus:

- Bien implementado el bonus! No hay errores malos en consola y es resposive.

Si tienes alguna duda, no olvides contactarme para hablar más de tu entregable.
Puedes hacer merge y continuar con los demas entregables.

Saludos

vinfante committed:
- Separated css from their styled components and imported them into their corresponding components
- Moved HomeScreen to pages, Added dummy Favorites page
- Cleaned up timeSince helper
- Centered gallery elements to view
